Week1/varXInteger.js

```diff
@@ -0,0 +1,7 @@
+//giving an integer value to variable x
+let x;
+console.log("the value of my variable x will be: my favorite number");
+console.log(x);
+x = 775;
+console.log("the value of x will be my favorite number: 775");
```

Comment:
You can remove `my favorite number` here, but the rest is good!

Reply:
done